README.md

```diff
@@ -23,38 +23,19 @@ You will need IntelliJ IDEA v2018.2.5 to run the latest version of this app, whi
 
 ### Setup
 #### Installing requirements
-If typing `python` gives you version 3.x):
 - `pip install -r requirements.txt`
```

Comment:
I think we should recommend running `pip3` instead of `pip`.  `pip` has the chance to run `pip2` depending on how the system is set up (`pip` either points to `pip3` or `pip2`), so specifying `pip3` as we do with `python3` in the `package.json` produces more reliable results.

Reply:
sure ill change it!